framework/wazuh/core/cluster/common.py

```diff
@@ -614,23 +859,39 @@ def error_receiving_file(self, taskname_and_error_details: str) -> Tuple[bytes,
         return b'ok', b'Error received'
 
     def get_node(self):
+        """Get basic information about the node.
+
+        Returns
+        -------
+        dict
+            Basic node information.
+        """
         return self.get_manager().get_node()
 
 
 def asyncio_exception_handler(loop, context: Dict):
-    """
-    Exception handler used in the protocol. Asyncio's default raises an exception and closes the transport.
-    The desired behaviour in this case is just to show the error in the logs.
-
-    :param loop: Event loop
-    :param context: A dictionary containing fields explained in
-                    https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.call_exception_handler
+    """Exception handler used in the protocol.
+
+    Asyncio's default raises an exception and closes the transport. The desired behaviour in
+    this case is just to show the error in the logs.
+
+    Parameters
+    ----------
+    loop : Asyncio event loop object
+        Event loop
+    context : dict
+        Dictionary containing fields explained in
+        https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.call_exception_handler.
     """
     logging.error(f"Unhandled exception: {context['exception']} {context['message']}\n"
                   ''.join(traceback.format_tb(context['exception'].__traceback__)))
 
 
 class WazuhJSONEncoder(json.JSONEncoder):
+    """
```

Comment:
1 LINE

Reply:
The format of this comment is because it describes a class, not a method. It is the same format used in the rest of the cluster classes.